src/services/bundlers/gelato.ts

```diff
@@ -61,6 +61,6 @@ export class Gelato extends Bundler {
   }
 
   public shouldReestimateBeforeBroadcast(network: Network): boolean {
-    return !!network.isOptimistic
+    return true
```

Comment:
as this is always `true`, we can remove `shouldReestimateBeforeBroadcast` check in all conditions we are using it